src/applications/check-in/day-of/tests/e2e/pages/Confirmation.js

```diff
@@ -45,6 +45,12 @@ class Confirmation {
       .and('include.text', 'Go to another appointment');
   };
 
+  validateConfirmationAlert = () => {
+    cy.get('[data-testid="confirmation-alert"]')
+      .invoke('text')
+      .should('have.length.gt', 0);
```

Comment:
This seems to work well. Curious about the 0, what is this asserting? Also the `gt`.

Nothing to change, just curious about it. For testing that things exist, we usually just do something like `.should('be.visible')`

Reply:
It's testing that there is text inside the alert. `be.visible` could pass with an empty alert.

Reply:
ah that makes sense, thanks for the info. We could probably use this to make some of our other similar tests more robust if we do a testing refactor in the future.